spec/hydra/works/models/concerns/generic_file/virus_check_spec.rb

```diff
@@ -7,8 +7,12 @@ class FileWithVirusCheck < ActiveFedora::Base
       include Hydra::Works::GenericFile::VirusCheck
     end
     class ClamAV
-      def self.instance; @instance ||= ClamAV.new; end
-      def scanfile(path); puts "scanfile: #{path}"; end
+      def self.instance
+        @instance ||= ClamAV.new
+      end
+      def scanfile(path)
+        puts "scanfile: #{path}"
```

Comment:
I think I put this in -- do we want to kill this puts as long as we're cleaning all this up?